### PR TITLE
Update readme to include graalpy and ol9 as new base image.

### PR DESCRIPTION
**Pull request by ierguiti**
Update readme to include graalpy and ol9 as new base image.

### CODE REVIEW

**Comment by ezzarghili** [comment]
Please use meaningful commit messages.

File: README.md [hunk 1]
```diff
@@ -14,21 +14,22 @@ All images support the installation of extra features. The following container i
 | native-image | A size compact GraalVM Community container image with the [Native Image](https://www.graalvm.org/reference-manual/native-image) support.                   | RPM-based<sup>1</sup> |
 | truffleruby  | A size compact GraalVM Community container image with the Ruby runtime. It uses a standalone build of [TruffleRuby](https://github.com/oracle/truffleruby/releases). | RPM-based<sup>1</sup> |
 | nodejs       | A size compact GraalVM Community container image with the [Node.js runtime](https://www.graalvm.org/reference-manual/js/NodeJS/).                          | RPM-based<sup>1</sup> |
+| graalpy       | A size compact GraalVM Community container image with the [GraalPy runtime](https://www.graalvm.org/22.2/reference-manual/python/).                          | RPM-based<sup>1</sup> |
```
Comment (olyagpl) [inline README.md]: Please use the link without a version: `https://www.graalvm.org/reference-manual/python/`
The empty version URL will always redirect to the latest feature release, for example, `https://www.graalvm.org/reference-manual/python/` will go to `https://www.graalvm.org/22.3/reference-manual/python/`.

File: README.md [hunk 2]
```diff
@@ -14,21 +14,22 @@ All images support the installation of extra features. The following container i
...
 
-**1**: RPM-based GraalVM Community container images are based on GraalVM components RPMs that are available for Oracle Linux 7 and Oracle Linux 8. Similar to any other available packages, you can install these components using `yum` on Oracle Linux 7 or `microdnf` on the Oracle Linux 8 based images.
+**1**: RPM-based GraalVM Community container images are based on GraalVM components RPMs that are available for Oracle Linux 7, Oracle Linux 8 and Oracle Linux 9. Similar to any other available packages, you can install these components using `yum` on Oracle Linux 7 or `microdnf` on the Oracle Linux 8 and Oracle Linux 9 based images.
```
Comment (olyagpl) [inline README.md:19]: Punctuation:

**1**: RPM-based GraalVM Community container images are based on GraalVM components RPMs that are available for Oracle Linux 7, Oracle Linux 8, and Oracle Linux 9. Similar to any other available packages, you can install these components using `yum` on Oracle Linux 7 or `microdnf` on the Oracle Linux 8 and Oracle Linux 9 based images.

**Review by olyagpl — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ierguiti) [reply README.md]: Thanks for the explanation

**Pull request merged**